packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -929,6 +929,13 @@ mixin WidgetInspectorService {
     );
 
     errorJson['errorsSinceReload'] = _errorsSinceReload;
+    errorJson['renderedErrorText'] = TextTreeRenderer(
```

Comment:
make sure renderedErrorTest is truncated is this is not the first error since reload. If you look at the default onError implementation in Futter it should do that.